addons/udes_stock/models/stock_quant_package.py

```diff
@@ -117,6 +118,25 @@ def has_same_content(self, other):
         return frozenset(self._get_all_products_quantities().items()) == \
                frozenset(other._get_all_products_quantities().items())
 
+    def mls_can_fulfil(self, mls):
+        """Returns mls which the package can fulfil if the product_qty of the
```

Comment:
Returns mls which the package can fulfill. If the product_qty of the mls is larger than in the package (i.e. in self) the mls will be split.